cypress/e2e/share-link.js

```diff
@@ -51,7 +51,18 @@ describe('Public sharing of office documents', function() {
 							cy.spy(win, 'postMessage').as('postMessage')
 						},
 					})
-					cy.waitForCollabora(true)
+
+					cy.waitForCollabora()
+					cy.get('@postMessage', { timeout: 20000 }).should(spy => {
+						const calls = spy.getCalls()
+						const findMatchingCall = calls.find(call => call.args[0].indexOf('"MessageId":"App_LoadingStatus"') !== -1)
+						if (!findMatchingCall) {
+							return expect(findMatchingCall).to.not.be.undefined
+						}
+						const object = JSON.parse(findMatchingCall.args[0])
+						expect(object.Values).to.have.property('Status', 'Initialized')
+					})
```

Comment:
We might want to move this into a cypress command that can be reused in the future. So we can assert post messages in other tests as wel. For now fine to just keep it as it is i think.